### ISSUE TITLE
_ treated as special character, but included in following word

### ISSUE THREAD

**Issue by ceejsmith** [lukeapage/node-markdown-spellcheck#59]
I recently wrote a blog post for Scott Logic where I mentioned a file called Docker_gateway. mdspell rejected it, and the solution was to add _gateway to .spelling.
If _ is being treated as a special character, it shouldn't be included in words either side of it.

**Comment by lukeapage** [comment]
Its underline in markdown, but possibly not stripping it because there is
no matching (or maybe markdown requires whitespace before?)

**Comment by ColinEberhardt** [comment]
Ah, good point - perhaps it would be better to backtick that text anyway? i.e. `Docker_gateway`

**Comment by ceejsmith** [comment]
Yes, it might be better to backtick it. I had a look at the stack trace for the error being thrown and dug into the code, but it'd take me longer than I've got to come up with a PR. I've got to get a bench project going! Nice work by the way Luke.
